### ISSUE TITLE
compile error - libtu/locale.h:20:10: fatal error: 'libintl.h' file not found

### ISSUE THREAD

**Issue by mpc-ucl** [raboof/notion#335]
Hi. I'm trying to compile notion and am running into the following problem: 

Qd:~/bin/notion% make
>> pkg-config found Lua 5.3 (available: lua).
>> Lua 5.3 binary is /opt/local/bin/lua and luac is /opt/local/bin/luac
set -e; for i in libmainloop libtu libextl mod_tiling mod_query mod_menu mod_dock mod_sp mod_sm de mod_mgmtmode mod_statusbar mod_xkbevents mod_notionflux ioncore notion etc utils man po contrib/scripts; do /Applications/Xcode.app/Contents/Developer/usr/bin/make -C $i; done
cc -g -Os -W -Wall -pedantic -DCF_XFREE86_TEXTPROP_BUG_WORKAROUND -DHAVE_X11_XFT -DHAVE_X11_BMF -I.. -I.. -I..  -DHAS_SYSTEM_ASPRINTF=1 -DNOTION_RELEASE='"-n 4.0.2-20-ge98a4dff -n +20210415-2243"' -D_XOPEN_SOURCE -D_XOPEN_SOURCE_EXTENDED -std=c99 -DCF_HAS_VA_COPY -MMD -c select.c -o select.o
cc -g -Os -W -Wall -pedantic -DCF_XFREE86_TEXTPROP_BUG_WORKAROUND -DHAVE_X11_XFT -DHAVE_X11_BMF -I.. -I.. -I..  -DHAS_SYSTEM_ASPRINTF=1 -DNOTION_RELEASE='"-n 4.0.2-20-ge98a4dff -n +20210415-2243"' -D_XOPEN_SOURCE -D_XOPEN_SOURCE_EXTENDED -std=c99 -DCF_HAS_VA_COPY -MMD -c defer.c -o defer.o
In file included from defer.c:20:
../libtu/locale.h:20:10: fatal error: 'libintl.h' file not found
#include <libintl.h>
         ^~~~~~~~~~~
1 error generated.
make[1]: *** [defer.o] Error 1
make: *** [subdirs] Error 2

**Comment by raboof** [comment]
Interesting - libtinl.h seems part of libc which should be pretty common. What distribution are you on?

**Comment by wilhelmy** [comment]
since it says Xcode.app in there I'm guessing OS X

**Comment by raboof** [comment]
Ah, I don't have any experience with X11 on OSX - contributions in this area are definitely welcome but be aware you might be in unchartered waters :)

**Comment by raboof** [comment]
Did you see #230?

**Comment by mpc-ucl** [comment]
Yes, thank you, I see. I admit I've successfully run the same compiled version of ion since around 2006 or so, but am now having to recompile due to osx upgrade to 64 bit only. I have not managed to recompile since then, so this might indeed be an issue that has been around for a while.

**Comment by wilhelmy** [comment]
`libintl.h` seems to be part of gettext... can you find that file anywhere in your system includes? Does `pkg-config --cflags gettext` print anything for you?

**Comment by wilhelmy** [comment]
https://github.com/EOSIO/eos/issues/2028 mentions they installed gettext via brew.
I guess we should mention gettext as a dependency on OSX in our docs, in case it builds okay otherwise.
